test/units/testsuite-65.sh

```diff
@@ -139,6 +139,16 @@ systemd-analyze cat-config systemd/system.conf systemd/journald.conf >/dev/null
 systemd-analyze cat-config systemd/system.conf foo/bar systemd/journald.conf >/dev/null
 systemd-analyze cat-config foo/bar
 
+if [[ -v ASAN_OPTIONS ]]; then
```

Comment:
Since you went with `-v`, you'll need to invert the condition:

```suggestion
if [[ ! -v ASAN_OPTIONS ]]; then
```

Reply:
Duh.